docs/governance/community-guidelines/releases.md

```diff
@@ -0,0 +1,44 @@
+# Releases
+
+This document is a [community guideline].
```

Comment:
Links are giving me a 404 but that might just be the GitHub preview thing

Reply:
Works for me locally (i.e. it opens the correct page in VS code when clicking on it) so I presume it is a preview thing.

Reply:
Yeah - link-checker is still showing the same amount of fails as before so should be good 👍